coreneuron/nrnconf.h

```diff
@@ -78,7 +78,7 @@ extern void* emalloc(size_t size);
 extern void* ecalloc(size_t n, size_t size);
 extern void* erealloc(void* ptr, size_t size);
 extern void* emalloc_align(size_t size, size_t alignment);
-extern void* ecalloc_align(size_t n, size_t alignment, size_t size);
+extern void* ecalloc_align(size_t n, size_t size, size_t alignment);
```

Comment:
we changed this API. could you check if we updated all previous usages with correct params? Just to be sure.

Reply:
I checked every instance of `ecalloc_align` and it seems that all the calls were updated to the new signature.

Was this just a safety check or did you see anything weird or wrong?